spk/gerrit/Makefile

```diff
@@ -0,0 +1,54 @@
+SPK_NAME = gerrit
+SPK_VERS = 2.11.4
+SPK_REV = 1
+SPK_ICON = src/gerrit.png
+DSM_UI_DIR = app
+
+DEPENDS = cross/busybox
+SPK_DEPENDS = "JavaManager:git"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Gerrit provides web based code review and repository management for the Git version control system.
+ADMIN_PROTOCOL = https
+ADMIN_PORT = 8091
+STARTABLE = yes
+RELOAD_UI = yes
+DISPLAY_NAME = Gerrit
+
+LICENSE = Apache 2.0
+HOMEPAGE = "https://gerritcodereview.com"
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT = src/dsm-control.sh
+FWPORTS = src/${SPK_NAME}.sc
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+
+#COPY_TARGET = nop
+
+POST_STRIP_TARGET = gerrit_extra_install
+
+BUSYBOX_CONFIG = usrmng
+ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
+
+GERRIT_WAR_URL="https://gerrit-releases.storage.googleapis.com/gerrit-${SPK_VERS}.war"
+
+include ../../mk/spksrc.spk.mk
+
+.PHONY: gerrit_extra_install
```

Comment:
This looks convoluted. Just create the dirs you need directly in one target, and move the files you have into them where needed.